src/index.js

```diff
@@ -0,0 +1,16 @@
+const tetrispiece = [
```

Comment:
It's more common to indicate that lists contain a plural
```suggestion
const tetrisPieces = [
```